docs/tutorial/tutorial.md

```diff
@@ -877,7 +877,7 @@ We also need to change it a little, since Game state is structured differently.
     squares[i] = this.state.xIsNext ? 'X' : 'O';
     this.setState({
       history: history.concat([{
-        squares: squares
+        squares,
```

Comment:
I could go either way, but I'd like to lean on the side of simple for a tutorial.